### ISSUE TITLE
A way to change the village building spawn rate

### ISSUE THREAD

**Issue by FarrazR007** [jtl-elisa/HUB-Better-Villages#135]
Hi is there a way to change certain village houses or building spawn rate like for example a way to change the spawn rate for the animal stable to always spawn each village loaded or to always have 2 of those church looking structure to spawn in each village?

**Comment by MalignantX** [comment]
I agree. I'm messing with the config. Only config options I can find:

int.villages.salt=10387312
int.villages.separation=20
int.villages.spacing=45

**Comment by tmslpm** [comment]
No, the configuration doesn’t support this. However, you can use datapacks if you want to control this kind of setting, please check the official wiki on this topic.

I don’t think we’ll be adding this kind of feature to the mod at the moment.
